cbaksik/HVD2
# Add Google Analytics

**Add Google Analytics**
* * *

**JIRA Ticket**: [LTSP-115](https://jira.huit.harvard.edu/browse/LTSP-115)

# What does this Pull Request do?
Inserts Google Analytics into <head> region of document

# How should this be tested?
* Spin up HOLLIS locally
* Inspect the `<head>` region and find the <script> tag that has inline code injected into it (will probably be towards the bottom of the section). Should look like this:
```
<script type="text/javascript">
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-52592218-13', 'auto');ga('send', 'pageview');
</script>
```

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
Tag (@ mention) interested parties
